src/pages/leeszaal/[slug].tsx

```diff
@@ -83,23 +90,17 @@ const ReadingRoomPage: NextPage = () => {
 		data && setMedia(data);
 	};
 
-	const onSearch = async (keywords: string[]) => {
+	const onSearch = async (values: SearchBarValue<true>) => {
 		if (!hasSearched) {
 			setHasSearched(true);
 		}
 
-		if (!keywords?.[0]) {
+		if (!values?.[0]) {
 			setMedia([]);
 			setQuery({ search: undefined });
 		} else {
 			await fetchMedia();
```

Comment:
Vermijd dit nieuwe executions van de `onSearch`-handler wanneer de `fetchMedia` request reeds in progress is?

Reply:
Deze `onSearch` gaat zwz nog herschreven moeten worden dus niet van wakker liggen 😄